src/server.js

```diff
@@ -1 +1,28 @@
-// TODO setup server
\ No newline at end of file
+/* eslint-disable no-console */
+// TODO setup server
+const express = require('express');
+
+const { sequelize } = require('../database/db.config');
+
+const app = express();
+
+require('dotenv').config();
+
+const port = process.env.PORT;
+
+// database connection using configuration in db.config.js
```

Comment:
Database connection checks should be done in the config file too. The purpose of the config file is to abstract everything about the database from the main entry file

Reply:
also port should have default port in case port is not set(best practice)
```
const port = process.env.PORT || 8080;
```

Reply:
Noted